MapleServer2/Types/DamageHandler.cs

```diff
@@ -46,7 +84,26 @@ public static DamageHandler CalculateDamage(SkillCast skill, IFieldActor source,
             return new(source, target, 0, HitType.Miss); // we missed
         }
 
-        bool isCrit = skill.IsGuaranteedCrit() || RollCrit(source, target, luckCoefficient);
+        double luckCoefficient = 1;
+        double attackDamage = 300;
+
+        if (source is IFieldActor<Player> player)
+        {
+            luckCoefficient = GetClassLuckCoefficient(player.Value.Job);
+
+            double bonusAttack = player.Stats[StatAttribute.BonusAtk].Total + 0.396 * player.Stats[StatAttribute.PetBonusAtk].Total;
```

Comment:
magic numbers, use an enum or so instead? constexpress LOL, but csharp doesnt have it.

Reply:
Just add this value in Constants.cs